docs/training/index.md

```diff
@@ -130,7 +130,7 @@ Streams can transport data between two operators in a *one-to-one* (or
 ## Timely Stream Processing
 
 For most streaming applications it is very valuable to be able re-process historic data with the
-same code that is used to process live data -- and to produce deterministic, consistent results,
+same code that is used to process live data and to produce deterministic, consistent results,
```

Comment:
A comma might be better than a dash, but some punctuation is needed to create a pause after "data". 

```suggestion
same code that is used to process live data, and to produce deterministic, consistent results,
```

Reply:
> A comma might be better than a dash, but some punctuation is needed to create a pause after "data".

Thx for your review. I've apply the suggestion with a new commit.